docs/preflight.md

```diff
@@ -17,7 +17,7 @@ preflight [url] [flags]
       --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
       --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
       --as-uid string                  UID to impersonate for the operation.
-      --cache-dir string               Default cache directory (default "/Users/xavpaice/.kube/cache")
+      --cache-dir string               Default cache directory (default "/Users/evans/.kube/cache")
```

Comment:
i know you didn't add this, but this is a very strange default....

Reply:
eek.  That's autogenerated documentations!

Reply:
I modified the docs. It should be OK cause for now we manually generate the docs and update them. When the process becomes automated we might need some `sed s/old/new/g` magic